src/renderer/components/Network/NetworkModal.vue

```diff
@@ -346,7 +347,29 @@ export default {
       this.form.nethash = this.network.nethash
       this.form.token = this.network.token
       this.form.symbol = this.network.symbol
-      this.form.version = this.network.version.toString()
+
+      // Temporary block to patch networks missing "version" - address prefix
+      if (this.network.version) {
+        this.form.version = this.network.version.toString()
+      } else {
+        const networkConfig = NETWORKS.find(network => network.id === this.network.id)
+
+        if (networkConfig) {
+          this.form.version = networkConfig.version.toString()
+        } else {
+          try {
+            ClientService.fetchNetworkConfig(this.form.server)
+              .then(network => {
+                console.log('fetchNetworkConfig')
+                this.form.version = network.version.toString()
+                this.$error('Address Version was missing - please check your wallets')
```

Comment:
Maybe move this to the locale file and remove debugging.

Reply:
Yes you're right 🤔 will do it a bit later on. Thanks 👌

Reply:
Done @luciorubeens